patches/@vercel+og+0.5.8.patch

```diff
@@ -13,35 +13,34 @@ index 5187f88..c4a1c41 100644
  
  // src/emoji/index.ts
  var U200D = String.fromCharCode(8205);
-@@ -18809,18 +18809,18 @@ async function render(satori, resvg, opts, defaultFonts, element) {
+@@ -18809,18 +18809,15 @@ async function render(satori, resvg, opts, defaultFonts, element) {
  // src/index.edge.ts
  var initializedResvg = initWasm(resvg_wasm);
  var initializedYoga = initYoga(yoga_wasm).then((yoga2) => Ll(yoga2));
 -var fallbackFont = fetch(new URL("./noto-sans-v27-latin-regular.ttf", import.meta.url)).then((res) => res.arrayBuffer());
-+// var fallbackFont = fetch(new URL("https://fonts.gstatic.com/s/notosans/v28/o-0IIpQlx3QUlC5A4PNr6zRF.ttf", import.meta.url)).then((res) => res.arrayBuffer());
```

Comment:
do we need to do anything to maintain this patch? if the `@vercel/og` version updates will this patch still be applied, or will we lose it and the test starts failing again?

Reply:
because we have @vercel/og pinned through yarn.lock, we'd need to actively take any new updates, at which point we would need to make a new patch for the new version. Without any patch the tests would fail, so there should be a blocker to anyone just updating the package without also updating the patch.